responses/sk/HassTurnOff.yaml

```diff
@@ -2,8 +2,10 @@ language: sk
 responses:
   intents:
     HassTurnOff:
-      default: Turned off {{ slots.name }}
-      lights_area: Turned off lights in {{ slots.name }}
+      # musi sa vyriesit sklonovanie, navrhujem pouzivat "Vypol som | Zatvaram ....."
+      default: "Vypínam {{ slots.name }}"
+      lights_area: "Svetlá v {{ slots.name }} boli zhasnuté"
       fans_area: Turned off fans in {{ slots.name }}
```

Comment:
tak isto (snad jsem to napsal správně :D) slots.area

Reply:
jo je to area, opravim